src/docs/background/13-how-payment-intents-work/index.md

```diff
@@ -0,0 +1,179 @@
+2. Customer uses the PaymentIntent data to complete any steps
+   necessary to authenticate and authorize the payment.
+3. The transaction can proceed only after customer has authorized (if required)
+   the payment. The PaymentIntent is confirmed, resulting in a Charge being
+   preauthorized.
+4. Transaction flow continues as usual onwards.
+
+![Automatic PaymentIntent flow](./automatic_confirmation_flow.png)
+
+For technical implementation of PaymentIntents, Stripe offers two approaches -
+[manual or automatic confirmation
+flow](https://stripe.com/docs/payments/payment-intents#one-time-payments). We
+have implemented the support for the automatic flow. In practice, the Flex
+transaction engine models the automatic flow with two transitions. First transition
+creates the PaymentIntent (Step 1.) and second transition will validate and
+mark it confirmed in Flex (Step 3.). Between these steps, the automatic flow
+pushes the responsibility of authenticating, authorizing and confirming the
+payment in Stripe to the client application (Step 2.). More information on the
+Step 2. can be found in this [chapter](#required-changes-in-the-client).
+
+## Actions related to PaymentIntents
+
+The following actions can be attached to a transaction process in order to
+implement PaymentIntent flow and are already present in the default flows.
+
+### stripe-create-payment-intent
+
+> Params:
+> - paymentMethod: optional, the Stripe id of paymentMethod
+>
+
+Creates a PaymentIntent. You can optionally pass in a [paymentMethod](https://stripe.com/docs/payments/payment-methods) id, or
+attach a paymentMethod later to the transaction during the validation and
+confirmation in the client by using Stripe Elements. The latter is the
+recommended way and is covered in the [implementation guide](#implementing-the-automatic-paymentintent-flow).
+
+### stripe-confirm-payment-intent
+
+Validates that the transaction has a PaymentIntent created and verifies via
+Stripe API that the PaymentIntent status is `requires_capture` or
+`requires_confirmation`. Confirms the PaymentIntent in Stripe, if needed, and
+marks the payment as preauthorized in Flex.
+
+### stripe-capture-payment-intent
+
+Captures a confirmed PaymentIntent. 
+
+### stripe-refund-payment
+
+Either cancels an unconfirmed PaymentIntent or refunds the related captured charge.
+
+## Required actions in the client
+
+The required actions in the client are related to authentication and
+confirmation. You need to be able to handle potential authentication steps
+enforced by regulation or banks. After authentication, the client needs to
+obtain PaymentIntent data from the transaction and use that to confirm the
+payment.
+
+Potential other changes are first enforcing [3D Secure Card
+Payments](https://stripe.com/docs/payments/3d-secure) through stripe [Radar
+rules](https://stripe.com/docs/payments/3d-secure#three-ds-radar) and then
+handling the necessary authentication steps.
```

Comment:
Language suggestion:

> In case you wish to enforce 3D Secure Card Payments for cards that support 3DS, in addition to supporting payment authentication in your client app, you may need to update your Stripe Radar rules.